src/pybind/cephfs/cephfs.pyx

```diff
@@ -163,7 +163,7 @@ class OSError(Error):
         self.strerror = strerror
 
     def __str__(self):
-        return '[Errno {0}] {1}'.format(self.errno, self.strerror)
+        return '{0}: {1} [Errno {2}]'.format(self.strerror, os.strerror(self.errno), self.errno)
```

Comment:
What's the difference between `self.strerror` and `os.strerror(self.errno)`. I couldn't see what `self.strerror` printed in your tracker ticket. So I'm confused.

Reply:
For example consider mkdir error message for attempting to create a directory with existing file name 'y'.
Then ```self.strerror``` stores the following string
```
error in mkdir 'b'y'
```
But ```os.strerror(self.errno)``` has standard error message instead of user-defined message.
```
File exists
````
The final error message would be ```error in mkdir y: File exists [Errno 17]```
Without this patch it is ```[Errno 17] error in mkdir 'b'y'```

Reply:
got it thanks